mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import cn from "classnames";
+import styles from "./Input.module.scss";
+
+
+interface InputProps {
+  type?: HTMLInputTypeAttribute;
+  placeholder?: string;
+  className?: string | Array<string>;
+  inputMode?: "none" | "text" | "tel" | "url" | "email" | "numeric" | "decimal" | "search" | undefined;
+  disabled?: boolean;
+  required?: boolean;
+}
+
+export const Input: FC<InputProps> = ({
+  type = "text",
+  placeholder = "",
+  className = "",
+  disabled = false,
+  required = false,
+  inputMode,
+}) => {
+  return (
+    <div className={styles.input__wrapper}>
+      <label
+        className={styles["visually-hidden"]}
+      >
+        {placeholder}
+      </label>
+      <input type={type}
```

Comment:
could you put a `type` prop on the next line? If linter didn't do it for you could you adjust it? please?